### ISSUE TITLE
Docker: `RUN apt-get install -y python3-pip=20.0.2-5ubuntu1.6` fails

### ISSUE THREAD

**Issue by maxkratz** [google-deepmind/dqn_zoo#22]
Currently, the Docker build of [the script](https://github.com/google-deepmind/dqn_zoo/blob/master/run.sh#L61) fails, because of `Version '20.0.2-5ubuntu1.6' for 'python3-pip' was not found`.

```
uild image with tag 'dqn_zoo:latest' and run tests
[+] Building 710.5s (9/33)                                                                                                                   docker:default
 => [internal] load .dockerignore                                                                                                                      0.0s
 => => transferring context: 2B                                                                                                                        0.0s
 => [internal] load build definition from Dockerfile                                                                                                   0.0s
 => => transferring dockerfile: 2.02kB                                                                                                                 0.0s
 => [internal] load metadata for docker.io/nvidia/cuda:11.1.1-cudnn8-devel-ubuntu20.04                                                                 1.5s
 => [auth] nvidia/cuda:pull token for registry-1.docker.io                                                                                             0.0s
 => [internal] load build context                                                                                                                      0.0s
 => => transferring context: 332.48kB                                                                                                                  0.0s
 => [ 1/28] FROM docker.io/nvidia/cuda:11.1.1-cudnn8-devel-ubuntu20.04@sha256:5b751d1720c635534bac0ffd4d77ddf21d5f1f814d57b72ff361f2700c94f712       669.4s
 => => resolve docker.io/nvidia/cuda:11.1.1-cudnn8-devel-ubuntu20.04@sha256:5b751d1720c635534bac0ffd4d77ddf21d5f1f814d57b72ff361f2700c94f712           0.0s
 => => sha256:5b751d1720c635534bac0ffd4d77ddf21d5f1f814d57b72ff361f2700c94f712 2.84kB / 2.84kB                                                         0.0s
 => => sha256:d694acee51c41c5404c7d37836b557b19efca2d05479e00ddd011865931da736 10.79MB / 10.79MB                                                       2.9s
 => => sha256:56e0351b98767487b3c411034be95479ed1710bb6be860db6df0be3a98653027 27.51MB / 27.51MB                                                       6.3s
 => => sha256:eceae4e0b416410cea1ffb811ddebd89c1884faaad412af5ec1788dbd5a19b16 7.94MB / 7.94MB                                                         3.6s
 => => sha256:be08d39904693fc307667f0726e1fd6a0b395b7d6944e14108843e01d5a89399 17.16kB / 17.16kB                                                       0.0s
 => => sha256:dd9eef69d4c16798a3fda6b9b4f9b8c22711d1e23dc7703c718a54511a628842 186B / 186B                                                             3.1s
 => => sha256:d2351b6b3529e61817fe5ccf29f1f072a9c5777c26541ef431f0d35bf662fadc 6.88kB / 6.88kB                                                         3.3s
 => => sha256:148309786e019674f606169af93cc6b1decc502616d79603df421ca35c6a8441 1.51GB / 1.51GB                                                       626.4s
 => => sha256:30431a42f9c525d046c48fced782d57744ee19f5b2a000e66220562fef90d27a 61.24kB / 61.24kB                                                       3.8s
 => => sha256:52512d7fe6e9910eabefc828412521131a955ed3b7414ffd3a1a866ce78a93eb 1.68kB / 1.68kB                                                         4.0s
 => => sha256:c562fd0c9fc50dd01a438b46afb955cb128e0a4de45b3c23ef3a8e8394018247 1.52kB / 1.52kB                                                         4.2s
 => => sha256:15b589b8315d98dc66061c4a23144a0e86d0b0d50e177459f42b5fbe31aaa7a1 1.60GB / 1.60GB                                                       606.8s
 => => extracting sha256:56e0351b98767487b3c411034be95479ed1710bb6be860db6df0be3a98653027                                                              0.4s
 => => sha256:4b439651229987f5111ebca8fae3aa7c3b7c7d0925e997c9aad87fa0f1bc7c44 84.08kB / 84.08kB                                                       6.7s
 => => sha256:40b57d883c72754db14f6ffcc6aee524808fa30cb36f4cc64519796eb1cddb6c 1.57GB / 1.57GB                                                       297.3s
 => => extracting sha256:eceae4e0b416410cea1ffb811ddebd89c1884faaad412af5ec1788dbd5a19b16                                                              0.1s
 => => extracting sha256:d694acee51c41c5404c7d37836b557b19efca2d05479e00ddd011865931da736                                                              0.1s
 => => extracting sha256:dd9eef69d4c16798a3fda6b9b4f9b8c22711d1e23dc7703c718a54511a628842                                                              0.0s
 => => extracting sha256:d2351b6b3529e61817fe5ccf29f1f072a9c5777c26541ef431f0d35bf662fadc                                                              0.0s
 => => extracting sha256:148309786e019674f606169af93cc6b1decc502616d79603df421ca35c6a8441                                                              8.1s
 => => extracting sha256:30431a42f9c525d046c48fced782d57744ee19f5b2a000e66220562fef90d27a                                                              0.0s
 => => extracting sha256:52512d7fe6e9910eabefc828412521131a955ed3b7414ffd3a1a866ce78a93eb                                                              0.0s
 => => extracting sha256:c562fd0c9fc50dd01a438b46afb955cb128e0a4de45b3c23ef3a8e8394018247                                                              0.0s
 => => extracting sha256:15b589b8315d98dc66061c4a23144a0e86d0b0d50e177459f42b5fbe31aaa7a1                                                             10.0s
 => => extracting sha256:4b439651229987f5111ebca8fae3aa7c3b7c7d0925e997c9aad87fa0f1bc7c44                                                              0.0s
 => => extracting sha256:40b57d883c72754db14f6ffcc6aee524808fa30cb36f4cc64519796eb1cddb6c                                                             15.2s
 => [ 2/28] RUN apt-get update                                                                                                                        32.1s
 => [ 3/28] RUN apt-get install -y python3.9=3.9.5-3ubuntu0~20.04.1                                                                                    6.1s 
 => ERROR [ 4/28] RUN apt-get install -y python3-pip=20.0.2-5ubuntu1.6                                                                                 1.2s 
------                                                                                                                                                      
 > [ 4/28] RUN apt-get install -y python3-pip=20.0.2-5ubuntu1.6:                                                                                            
0.459 Reading package lists...                                                                                                                              
0.990 Building dependency tree...                                                                                                                           
1.102 Reading state information...                                                                                                                          
1.114 E: Version '20.0.2-5ubuntu1.6' for 'python3-pip' was not found
------
Dockerfile:13
--------------------
  11 |     
  12 |     # Install pip.
  13 | >>> RUN apt-get install -y python3-pip=20.0.2-5ubuntu1.6
  14 |     RUN python3.9 -m pip install --upgrade pip==22.1.2
  15 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c apt-get install -y python3-pip=20.0.2-5ubuntu1.6" did not complete successfully: exit code: 100
```

This "fixes" the error (at least for build the image, I did not test it, yet):
```
diff --git a/Dockerfile b/Dockerfile
index aff620f..deeb3b7 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -10,7 +10,7 @@ ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get install -y python3.9=3.9.5-3ubuntu0~20.04.1
 
 # Install pip.
-RUN apt-get install -y python3-pip=20.0.2-5ubuntu1.6
+RUN apt-get install -y python3-pip
 RUN python3.9 -m pip install --upgrade pip==22.1.2
 
 # Install wget and unrar to download and extract ROMs.
```

**Comment by jqdm** [comment]
Thanks for reporting this, will send out a fix tomorrow.

**Comment by jqdm** [comment]
Fixed by 8728c674420725f49f6dec9b893f9420fd5cd2ef.
